### PR TITLE
New rule: Google Calendar invite linking to an open redirect from an untrusted freemail sender

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

File: detection-rules/link_gcal_invite_open_redirect.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    or sender.email.domain.root_domain not in $high_trust_sender_root_domains
+  )
+  and (
+    not profile.by_sender().solicited
+    or (
+      profile.by_sender().any_messages_malicious_or_spam
+      and not profile.by_sender().any_false_positives
+    )
+  )
+  and not profile.by_sender().any_false_positives
```
Comment (zoomequipd) [inline detection-rules/link_gcal_invite_open_redirect.yml:40]: IIRC there was some discussion about how any_false_positives isn't suppose to be used by itself. 

convo here: https://github.com/sublime-security/sublime-rules/pull/1773#discussion_r1713834810

**Comment by aidenmitchell** [comment]
No hits in shared samples

File: detection-rules/link_gcal_invite_open_redirect.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+severity: "high"
+source: |
+  type.inbound
+  and strings.iends_with(subject.subject, "has shared a calendar with you")
+  and strings.ilike(headers.message_id, "<calendar*@google.com>")
+  and sender.email.domain.domain in $free_email_providers
+  and any(body.links,
+          any(.href_url.rewrite.encoders, strings.icontains(., "open_redirect"))
+          or (
+            any(beta.linkanalysis(.).final_dom.links,
```
Comment (zoomequipd) [inline detection-rules/link_gcal_invite_open_redirect.yml:13]: update to non-beta version?

**Review by zoomequipd — CHANGES_REQUESTED** [review]

**Review by zoomequipd — APPROVED** [review]

**Pull request merged**